pkg/dev/dev.go

```diff
@@ -53,8 +59,49 @@ func (o *DevClient) Start(devfileObj parser.DevfileObj, platformContext kubernet
 	if err != nil {
 		return err
 	}
-	klog.V(4).Infoln("Successfully created inner-loop resourcs")
-	fmt.Fprintf(out, "\nYour application is running on cluster.\n ")
+	klog.V(4).Infoln("Successfully created inner-loop resources")
+
+	return nil
+}
+
+func (o *DevClient) Cleanup() error {
+	var err error
+	return err
+}
+
+func (o *DevClient) SetupPortForwarding(pod *corev1.Pod, portPairs []string, errOut io.Writer) error {
```

Comment:
This method could be part of the kclient package IMO

Reply:
Makes sense. How should it be called? I mean, is it fine to call it right from the CLI layer in `pkg/odo/cli/dev/dev.go`? I'm asking because when CLI layer calls it directly, it's not going through the business layer.